sklearn/preprocessing/data.py

```diff
@@ -2919,9 +2923,10 @@ class UnaryEncoder(BaseEstimator, TransformerMixin):
     sparse : boolean, default=False
         Will return sparse matrix if set True else will return an array.
 
-    handle_unknown : str, 'error' or 'ignore'
-        Whether to raise an error or ignore if a unknown ordinal feature is
-        present during transform.
+    handle_greater : str, 'error' or 'clip'
+        Whether to raise an error or clip if a greater ordinal feature value is
```

Comment:
"Whether to raise an error or clip if an ordinal feature >= n_values is passed in" is more precise, I think.

I think the same behaviour should happen in fit if `n_values` is specified.